doc/source/blisko.md

```diff
@@ -0,0 +1,213 @@
+# Blisko.App
+
+Support for schedules provided by [Blisko](https://blisko.co/).
+
+Blisko app supports mutliple voivodeships and cities. 
+
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: blisko
+      args:
+        regionId: 112
+        formattedId: "32:11:01:2:0774204:42719:32"
+```
+
+### Configuration Variables
+
+**regionId**  
+*(string) (required)*
+
+**formattedId**  
+*(string)(required)*
+
+All arguments must be provided.
+
+## Example
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: blisko
+      args:
+        regionId: 112
+        formattedId: "32:11:01:2:0774204:42719:32"
+```
+
+## How to get the source arguments
+
+You have to provide: 
+* `regionId` - An ID of owner of Blisko instance for your city.
+* `formattedId` - An ID of owner of Blisko instance for your city.
+
+### `regionId`
+
+
+Region ID can be either queried from `Blisko_searcher.py` helper script or find in the list below:
+
+#### Using Blisko_searcher.py 
+
+```
+$ python custom_components/waste_collection_schedule/waste_collection_schedule/service/Blisko_searcher.py --dumpregions
```

Comment:
People really are not that good at running those scripts, so maybe adding the git clone command could help